test/acceptance/dashboard_test.clj

```diff
@@ -107,8 +102,10 @@
   (t/testing "When the current user has already made Haggadot and goes to their dashboard, the Haggadot should be displayed in order from most recent to least recent"
     (create-haggadot haggadot "user1")
     (doto driver
-      (c/home->dashboard))
-    (let [titles (haggadot-titles (all-haggadot))]
+      (c/home->dashboard)
+      #_#(e/wait 9))
```

Comment:
Perhaps you don't need this comment...?

Reply:
I forgot to remove the additional wait. I'll take it out.